website/sidebars.json

```diff
@@ -12,15 +12,14 @@
       "overview",
       "let-binding",
       "primitives",
-      "type",
       "basic-structures",
+      "type",
```

Comment:
Does this make sure the slug stays the same (bookmarks).

Reply:
Yeah the URL is based entirely on the file name from what I can tell